HW03 Olivia Pura.Rmd

```diff
@@ -0,0 +1,119 @@
+---
+title: "HW03 Olivia Pura"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+To start, I'm loading in the ggplot2, readr, and dplyr packages.
+
+```{r packages, warning=FALSE, message=FALSE, echo=TRUE}
+library("ggplot2")
+library("readr")
+library("dplyr")
+```
+
+I will be working with the NYT COVID-19 dataset, "us-states.csv". I forked this dataset from the NYT repository and uploaded them into my own.
+
+```{r data}
+state_data <- read.csv("us-states.csv", stringsAsFactors = FALSE)
+```
+
+## Graph 1
+
+My first graph will evaluate the total cases and deaths over time in Illinois.
+
+```{r graph1, echo=TRUE}
+# filtering the dataset for Illinois data
+il_data <- filter(state_data, state == "Illinois")
+
+# formatting the date column using the as.Date() function for graph formatting
+il_data$date <- as.Date(il_data$date)
+
+# dividing the count of deaths and cases by 1,000 for cleaner axis
+il_data$cases <- il_data$cases/1000
+il_data$deaths <- il_data$deaths/1000
+
+# creating the basic column chart to display cases over time
+ggplot(il_data) +
+  geom_col(aes(x = date, y = cases), fill = "midnightblue", alpha = 0.6, width = 0.7) +
+  geom_col(aes(x = date, y = deaths), fill = "palevioletred1", alpha = 1, width = 0.7) + 
+ 
+# formating axis labels to make graph legible
+  scale_x_date(name = "Date", date_breaks = "2 weeks")  +
+  scale_y_continuous(name = "Count (thousands)", limits = c(0, 170), 
+                     breaks = seq(0, 170, 20),
+                     minor_breaks = seq(0, 170, 10)) +
+  
+# adding a title and subtitle
+  labs(title = "COVID-19 in Illinois", subtitle = "Total Number of Cases and Deaths Since First-Reported Case on January 24, 2020") +
+  
+# creating a legend
```

Comment:
That's a creative way to generate a legend for this plot. I really like the outcome.